tools/hasher/src/main/java/org/apache/shiro/tools/hasher/Hasher.java

```diff
@@ -255,10 +254,9 @@ public static void main(String[] args) {
                 } else {
                     iterations = DEFAULT_NUM_ITERATIONS;
                 }
-            } else {
-                //Iterations were specified, so add the iterations parameter:
-                parameters.put(SimpleHashProvider.Parameters.PARAMETER_ITERATIONS, iterations);
             }
+            //Iterations were specified, so add the iterations parameter:
```

Comment:
For me this not correct because we want to insure that the iterations parameter specified by the user is not too weak.